### PR TITLE
proovframe/fix module

### PR DESCRIPTION
**Pull request by vagkaratzas**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/proovframe/fix/main.nf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/proovframe:0.9.7--hdfd78af_1':
+        'biocontainers/proovframe:0.9.7--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(fa)
```
Comment (luisas) [inline modules/nf-core/proovframe/fix/main.nf:11]:
```suggestion
    tuple val(meta) , path(fa)
```

File: modules/nf-core/proovframe/fix/main.nf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    proovframe   \\
+        fix \\
+        ${args} \\
+        -o ${prefix}.fa  \\
+        ${fa} \\
+        ${tsv}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        proovframe: 0.9.7
```
Comment (luisas) [inline modules/nf-core/proovframe/fix/main.nf:34]: This one should be extracted from the tool directly, something like: https://github.com/nf-core/modules/blob/cb08035150685b11d890d90c9534d4f16869eaec/modules/nf-core/fastqc/main.nf#L49

File: modules/nf-core/proovframe/fix/main.nf [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    stub:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.fa
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        proovframe: 0.9.7
```
Comment (luisas) [inline modules/nf-core/proovframe/fix/main.nf:46]: same here

**Review by luisas — APPROVED** [review]
Just a minor comment about the version, otherwise LGTM!

File: modules/nf-core/proovframe/fix/main.nf (hunk 2 above)
Comment (vagkaratzas) [reply modules/nf-core/proovframe/fix/main.nf:34]: We tried really hard to make that happen..but really couldn't figure out a way..

Comment (vagkaratzas) [reply modules/nf-core/proovframe/fix/main.nf:34]: Ok think I found a way, brb :P

Comment (vagkaratzas) [reply modules/nf-core/proovframe/fix/main.nf]: Changed it for the proovframe/map module as well

**Pull request merged**